src/components/Explorer/ExplorerFormField.jsx

```diff
@@ -35,6 +35,19 @@ class ExplorerFormField extends React.Component {
           searchText,
         });
       }
+    } else if (this.autocomplete && this.autocomplete.state.searchText) {
```

Comment:
isn't this the same code as the other block?

Reply:
No , the previous block is called when the search text is not set. But here the search text is set when we pass from the url directly.

Reply:
yes, but if the inner code is the same, can't you just change the boolean condition to execute the same block instead of duplicating the code?

Reply:
Yes it is the same . I'll update it.